espanol/comandante/comandante.md

```diff
@@ -2,3 +2,4 @@ El comandante en jefe chuck norris, mas conocido como walker ranger de texas, es
 Al enterarse de dichos acontecimientos recomendo llamar al almirante tiger woods
 
 [Llamar al almirante..](almirante/almirante.md)
+[Llamar al coronel..](coronel/coronel.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.